unit test case TestDescribeServiceAccount probabilisticly fail

<!-- This form is for bug reports and feature requests ONLY! 

If you're looking for help check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
-->

**Is this a BUG REPORT or FEATURE REQUEST?**:

 /kind bug



**What happened**:
unit test fail randomly. 

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
```
cd k8s.io/kubernetes/pkg/printers/internalversion
go test -v -test.run TestDescribeServiceAccount
```
**Anything else we need to know?**:
```
# go test -v -test.run TestDescribeServiceAccount
=== RUN   TestDescribeServiceAccount
--- FAIL: TestDescribeServiceAccount (0.00s)
	describe_test.go:1694: expected : "Name:                bar\nNamespace:           foo\nLabels:              <none>\nAnnotations:         <none>\nImage pull secrets:  test-local-ref(not found)\nMountable secrets:   test-objectref (not found)\nTokens:              <none>\nEvents:              <none>\n"
		 but got output:
		 "Name:                bar\nNamespace:           foo\nLabels:              <none>\nAnnotations:         <none>\nMountable secrets:   test-objectref (not found)\nTokens:         <none>\nImage pull secrets:  test-local-ref (not found)\nEvents:              <none>\n"
FAIL
exit status 1
FAIL	k8s.io/kubernetes/pkg/printers/internalversion	0.082s

```
**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/pkg/printers/internalversion/describe.go
+++ b/pkg/printers/internalversion/describe.go
@@ -2273,7 +2273,8 @@ func describeServiceAccount(serviceAccount *api.ServiceAccount, tokens []api.Sec
 			mountHeader: mountSecretNames,
 			tokenHeader: tokenSecretNames,
 		}
-		for header, names := range types {
+		for _, header := range sets.StringKeySet(types).List() {
+			names := types[header]
 			if len(names) == 0 {
 				w.Write(LEVEL_0, "%s\t<none>\n", header)
 			} else {

